src/pages/settings/Profile/CustomStatus/StatusPage.js

```diff
@@ -86,7 +86,9 @@ function StatusPage({draftStatus, currentUserPersonalDetails}) {
             });
 
             User.clearDraftCustomStatus();
-            navigateBackToPreviousScreen();
+            InteractionManager.runAfterInteractions(() => {
+                navigateBackToPreviousScreen();
+            });
```

Comment:
Hey @perunt, could you please share why this `runAfterInteractions` was necessary here? It's causing some UX inconsistency: https://github.com/Expensify/App/issues/34886, so I was thinking if we could remove it, or it's absolutely necessary here?

Reply:
Maybe you could remember on what platform this was happening? I couldn't reproduce on the web even with the network set to "Slow 3G".